openmls/src/group/mls_group/mod.rs

```diff
@@ -422,6 +423,16 @@ impl MlsGroup {
     pub(crate) fn clear_pending_proposals(&mut self) {
         self.proposal_store.empty()
     }
+
+    /// Removes a specific proposal from the store and return it if it exists.
+    pub fn clear_pending_proposal(
```

Comment:
I think I would call this `remove` and not clear. "clear" is mostly used as in "clear everything" (see above).